app/build.gradle

```diff
@@ -24,7 +24,8 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    implementation 'com.google.android.gms:play-services:11.8.0'
```

Comment:
If you're replacing `play-services` with `play-services-wallet`, please update or remove the comment below as well.

Also note that we had appcompat in the past, but this caused problems with some partners who copied it from the sample, resulting in having the wrong version. I advise against including.